python/sbp/sbp2json.py

```diff
@@ -140,10 +149,34 @@ def floatstr(o, allow_nan=self.allow_nan,
             self.skipkeys, _one_shot)
         return _iterencode(o, 0)
 
-def get_jsonable(res):
-    if hasattr(res, 'to_json_dict'):
-        return res.to_json_dict()
-    return res
+
```

Comment:
What if the log has no MSG_UTC_TIME?  I feel like you might want to warn if LAST_UTC_TIME is never set by having received the message.